TowerNinja/Assets/Scripts/friendDmgHitbox.cs

```diff
@@ -50,7 +60,8 @@ private void OnTriggerEnter2D(Collider2D collision)
         
         if (collision.tag == "enemy")
```

Comment:
we can probably use the <object>.compareTag("<value>") method instead of string equals. My IDE gets annoyed when it's string equals, so it's probably better in some way.